admission-webhook/deploy/deploy-gmsa-webhook.sh

```diff
@@ -170,9 +170,21 @@ main() {
         effect: NoSchedule'
     fi
 
+    if [ -f "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt" ]; then
+        info 'using pod based authentication'
+        BUNDLE=$(cat /var/run/secrets/kubernetes.io/serviceaccount/ca.crt | base64 | tr -d '\n')
+    else
+        info 'using config file authentication'
+        BUNDLE=$($KUBECTL config view --raw --minify --flatten -o jsonpath='{.clusters[].cluster.certificate-authority-data}')
+    fi
+
+    if [[ -z "$BUNDLE" ]]; then
```

Comment:
not sure if this will ever happen

Reply:
if you didn't have a context set and you were not in a pod this *could* happen though unlikely